src/containers/Main.tsx

```diff
@@ -0,0 +1,113 @@
+import { inject, observer } from 'mobx-react';
+import React, { useCallback, useMemo } from 'react';
+import styled from 'styled-components';
+
+import { Button } from '../components/Button';
+import { Input } from '../components/Input';
+import { RepoItem } from '../components/RepoItem';
+import useInput from '../hooks/useInput';
+import Store from '../stores/store';
+
+interface Props {
+  store: Store;
+}
+
+const Main: React.FC<Props> = (props) => {
+  const { state, onChange } = useInput({ username: "" });
+
+  const { username } = state;
+
+  const { store } = props;
+
+  const repoItems = useMemo(() => {
+    return store.data.map((el) => <RepoItem {...el} />);
+  }, [store.data]);
+
+  const userInfo = useMemo(() => {
+    return (
+      store.username && (
+        <UserInfo>
+          <UserName>{store.username}</UserName>
+          <Count>{store.repoCount} Repositories</Count>
+          <Count>{store.totalStars} Stars</Count>
+        </UserInfo>
+      )
+    );
+  }, [store.username, store.repoCount, store.totalStars]);
+
+  const handleSearch = useCallback(async () => {
+    store.fetchData(username);
+  }, [store, username]);
+
+  const handleKeyUp = useCallback(
+    (e: React.KeyboardEvent<HTMLInputElement>) => {
+      if (e.keyCode === 13) {
+        handleSearch();
+      }
+    },
+    [handleSearch]
+  );
+
+  return (
+    <Container>
+      <Header>
+        <Title>GitStar Ranking</Title>
+        <Description>Unofficial GitHub Star ranking for users, organizations and repositories</Description>
+        <Input onChange={onChange} name="username" onKeyUp={handleKeyUp} />
+        <StyledButton onClick={handleSearch}>Search</StyledButton>
+      </Header>
+      {userInfo}
+      <RepoItems>{repoItems}</RepoItems>
+    </Container>
+  );
+};
+
+const Container = styled.div``;
+
+const Header = styled.div`
```

Comment:
시맨틱하게 header 어떤가여 ㅎㅎ 
```suggestion
const Header = styled.header`
```

Reply:
좋아요!

Reply:
근데 제가 생각하는 헤더는 약간 이런거였는데 이렇게 보니 제 변수 네이밍이 잘못되었다는 생각이 드네요. 그냥 위에있어서 Header로 지은거 같네요 ㅋㅋㅋㅋ

<img width="1666" alt="스크린샷 2020-04-15 오후 9 25 17" src="https://user-images.githubusercontent.com/38802280/79336971-a2e8ae00-7f5f-11ea-8c9d-1cade2441d67.png">